.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: node_js
+node_js:
+- '8'
+matrix:
+  include:
+  - os: linux
+    dist: xenial
+    sudo: required
+  - os: osx
+cache:
+  yarn: true
+notifications:
+  email: false
+script:
+- npm run build
+- npm run build:electron
+deploy:
+  provider: releases
+  api_key:
+    secure: H72TbvbYGaf8vjRZvTZuXQMDxiOIuYskVQJQrGBF6B7F748SwH6UIBBkscXD1rVE1c55WhmuWIGGUZi1++wlhBHylwmBwfqwcbVykMiRMHMU6l9PEMIvxMOCyVZlBxxo0g3ULOJnvoakGC6NCxrJAl4jLIvRzZHMGACzTNpTei8=
+  on:
+    repo: lexoyo/Silex
+    branch: electron-ci
+env:
+  global:
+    - GH_TOKEN=H72TbvbYGaf8vjRZvTZuXQMDxiOIuYskVQJQrGBF6B7F748SwH6UIBBkscXD1rVE1c55WhmuWIGGUZi1++wlhBHylwmBwfqwcbVykMiRMHMU6l9PEMIvxMOCyVZlBxxo0g3ULOJnvoakGC6NCxrJAl4jLIvRzZHMGACzTNpTei8=
```

Comment:
Put that in your secret Travis configuration and is this var in the `api_key`

Reply:
I don't know what it is... And it still does not deploy. Maybe you can give it a go
It builds for Linux and Mac which is already something good

Reply:
After reading the [manual for GitHub Releases](https://docs.travis-ci.com/user/deployment/releases/), I think you're missing 3 things:

- [ ] `skip_cleanup = true`
> Make sure you have skip_cleanup set to true, otherwise Travis CI will delete all the files created during the build, which will probably delete what you are trying to upload.
- [ ] `on.tags = true` because GH works like this and it will be a more cleaner
- [ ] `file` pointing to your executable